src/component/noteList/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import NoteItem from '../noteItem';
+
+export default class NoteList extends React.Component {
+  render() {
+    return (
+      <div> 
+          <ul className="noteList">
+            {
+              this.props.notes.map((note) => {
```

Comment:
you'll also want to pass down the note removing handler function here as well, this way each individual note will have the ability to delete itself